src/misc/ErrorBanner/ErrorBanner.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react';
+import { Button, Paper, Slide, Typography } from '@material-ui/core';
+import PropTypes from 'prop-types';
+import useStyles from './style';
+
+export const ErrorBanner = (props) => {
+  const classes = useStyles();
+  const { error, clearErrors, tooLongErrorMessage, dismissButtonText, secondButtonText, toggledButtonText } = props;
+  const [copyButtonText, setCopyButtonText] = useState(secondButtonText);
+  const copyToClipboard = (message) => {
+    navigator.clipboard.writeText(message).then(() => setCopyButtonText(toggledButtonText));
+  };
+  return (
+    <Slide direction="down" in={error != null} unmountOnExit>
+      <Paper square elevation={0} className={classes.errorContainer} data-cy="error-banner">
+        <div>
+          <Typography className={classes.errorTitle}>{error.status + ' ' + error.title}</Typography>
+          <Typography className={classes.errorText}>
+            {error.detail.length < 200 ? error.detail : tooLongErrorMessage}
+          </Typography>
+          <Typography className={classes.errorText}>{error.comment}</Typography>
+        </div>
+        <div className={classes.errorText}>
+          {error.detail.length > 199 && (
+            <Button
+              className={classes.errorButton}
+              size="small"
+              color="inherit"
+              variant="outlined"
+              onClick={() => copyToClipboard(error.detail)}
+            >
+              {copyButtonText}
+            </Button>
+          )}
+          <Button
+            className={classes.errorButton}
+            size="small"
+            color="inherit"
+            variant="outlined"
+            data-cy="dismiss-error-button"
+            onClick={clearErrors}
+          >
+            {dismissButtonText}
+          </Button>
+        </div>
+      </Paper>
+    </Slide>
+  );
+};
+
+ErrorBanner.propTypes = {
+  error: PropTypes.object.isRequired,
+  clearErrors: PropTypes.func.isRequired,
+  tooLongErrorMessage: PropTypes.string,
+  dismissButtonText: PropTypes.string.isRequired,
+  secondButtonText: PropTypes.string.isRequired,
+  toggledButtonText: PropTypes.string,
```

Comment:
In order to be consistent with the other UI components, you can regroup these string props in a single `labels` prop (for an example, you can check the [DontAskAgainDialog](https://github.com/Cosmo-Tech/webapp-component-ui/blob/main/src/dialogs/DontAskAgainDialog/DontAskAgainDialog.js#L94) component)